### PR TITLE
Fix Team set in PlayerReference being ignored for unplayable players

### LINKED ISSUE #8623: Team set in PlayerReference is ignored for unplayable players
E.g. https://github.com/abcdefg30/OpenRA/commit/6b393c3cceee8d136b4cd729ecf8c2626f120226 gives me
lockteam
(I forced the Soviets team to `2` actually.)

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/23553838-team-set-in-playerreference-is-ingnored-for-unplayable-players?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by abcdefg30**
Fixes #8623.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public GameInfoStatsLogic(Widget widget, World world)
 				}
 
 				var team = item.Get<LabelWidget>("TEAM");
-				var teamNumber = (client == null) ? 0 : client.Team;
+				var teamNumber = pp.PlayerReference.Team;
```
Comment (pchote) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs]: This will probably break team display for regular players.

Comment (abcdefg30) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs]: Ops, fixed.

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark:

**Comment by penev92** [comment]
:+1:

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/c238339dbd0964dfb8d0f3898c88d7ef4f0346fd...012b02ec4d3c7247676f8cbd23393689843cd6a5)

**Pull request merged**